pkg/kubelet/cm/pod_container_manager_linux.go

```diff
@@ -197,6 +197,7 @@ func (m *podContainerManagerImpl) Destroy(podCgroup CgroupName) error {
 		ResourceParameters: &ResourceConfig{},
 	}
 	if err := m.cgroupManager.Destroy(containerConfig); err != nil {
+		klog.Warningf("failed to delete cgroup paths for %v : %v", podCgroup, err)
```

Comment:
@tedyu I don't feel super strongly, but one advantage of logging this using `klog.V(3).Infof` would be that it is consistent with the logging statement on line 190. I don't have a ton of context on this part of the code base, but I do get the sense they are describing equally bad failure states... in other words, it seems sensible to me that they'd have the same log level.

Alternatively, I suppose we could change both to be a `Warning`.

Reply:
@mattjmcnaughton that was my initial rational around consistency. I believe this error and a number of others logged at debug are generally ok to ignore as they should be self-resolving through subsequent sync loops. You would probable only want to see them when you find a definitive problem case (like what I am trying to root cause).

Reply:
Gotcha! So just to be sure we're on the same page, are you planning on changing it to `klog.V(3).Infof`? Or is there a reason more important than consistency for it to be `klog.Warningf`?

Reply:
@mattjmcnaughton In the original PR I had it at v3 info and per the discussion with @tedyu I changed it to warn. I am on the side of having it an v3 info and am happy to change it back it everyone is happy with that per my explanation.